src/sql/src/statements/copy.rs

```diff
@@ -43,6 +120,36 @@ pub struct CopyDatabaseArgument {
     pub location: String,
 }
 
+impl CopyDatabaseArgument {
+    #[inline]
+    fn format_with(&self) -> String {
+        if self.with.map.is_empty() {
+            String::default()
+        } else {
+            let options = format_sorted_hashmap!(&self.with.map);
```

Comment:
No all options should be unparsed as is. This is the original motivation of https://github.com/GreptimeTeam/greptimedb/issues/3646. We'd like to redact sensitive options like `access_key_id` and `secret_access_key`.

Although we may do it in a follow-up PR, this patch should be open to that change instead of making the wrong assumption.